### ISSUE TITLE
Inaccurate documentation

### ISSUE THREAD

**Issue by dalcde** [time-rs/time#196]
Here it says if deprecated is enabled, then the `log` crate will be a dependency: https://github.com/time-rs/time/blob/ab09c479160423f616fde55d62c1c0a6009f3b0a/src/lib.rs#L44

However, upon inspection of `Cargo.toml`, this does not appear to be true.

**Comment by jhpratt** [comment]
Good catch! The dependency was removed after I'd already performed inspection of the docs. I've just fixed this on master.
